app/index.js

```diff
@@ -31,6 +31,15 @@ const strategy = new Auth0Strategy(
     return callback(null, profile._json);
   }),
 );
+// Original implementation in `passport-openidconnect` ignore options by
+// returning `{}`.
+//
+// `passport-auth0-openidconnect` is supposed to override it but it doesn't.
+//
+// See: https://github.com/siacomuzzi/passport-openidconnect/blob/master/lib/strategy.js#L338
+Auth0Strategy.prototype.authorizationParams = (options) => {
+  return options;
```

Comment:
Just realised, shouldn't this be `options => options`?

Reply:
You're right. I think I didn't use it already because originally I was using `Object.assign(options, {})` and the body of the function was a bit "long" to use the shorthand.

I wonder if I should be defensive here and `return options || {};` - Probably not worth it as we control the code and chance of not passing `options` are very thin.

(I'm trying to write some tests to have something like this [upstream](https://github.com/auth0/passport-auth0-openidconnect/issues/3) - in that case it makes more sense to be defensive)

Reply:
FYI: https://github.com/ministryofjustice/analytics-platform-rstudio-auth-proxy/pull/13/commits/558de3c6ea3025bfdf3fa72481b38b228864aeb3#diff-2018087f584c4398b5c3a23fc0e5f9dbR40